### ISSUE TITLE
Stolen ADA 

### ISSUE THREAD

**Issue by KarlJH1997** [Berry-Pool/nami-wallet#603]
I have had my ADA stolen from my wallet, anyway to get it back?

**Comment by Arond-Devv** [comment]
@KarlJH1997 Talk to Cardano Support on the [Live support page](https://developerscardanno.com) so that your issue can be looked into.

**Comment by KarlJH1997** [comment]
Doesnt seem like a legit website... seems like a scam version of https://developers.cardano.org/

**Comment by gabriellemon** [comment]
Yeah, those scam sites look to be rampant on here. I'm sorry but I don't think you'll be able to do anything about it. Hopefully you didn't loose too much.
